client/my-sites/plugins/plugin-action/plugin-action.jsx

```diff
@@ -31,8 +31,8 @@ class PluginAction extends Component {
 				ref={ this.disabledInfoLabelRef }
 				onClick={ this.handleAction }
 			>
-				{ this.props.label }
-				{ this.renderDisabledInfo() }
+				<span className="plugin-action__label-text">{ this.props.label }</span>
+				<span className="plugin-action__label-disabled-info">{ this.renderDisabledInfo() }</span>
```

Comment:
Just checking, where is the class `plugin-action__label-disabled-info` used?

Reply:
Now its being used to apply your suggestion: https://github.com/Automattic/wp-calypso/pull/67984/commits/29a1de4023ee3ad7043830c071d9b9bec3a0b319 :D